src/neo/SmartContract/Native/Oracle/OraclePolicyContract.cs

```diff
@@ -42,6 +42,7 @@ public OraclePolicyContract()
         internal override bool Initialize(ApplicationEngine engine)
         {
             if (!base.Initialize(engine)) return false;
+            if (GetPerRequestFee(engine.Snapshot) != 0) return false;
```

Comment:
It will cause bug here.UT can not be complete

Reply:
mmm. it should be empty during the `Initialize`